packages/hooks/src/useGetPasskeysList.ts

```diff
@@ -6,13 +6,14 @@ import { useStore } from '@deriv/stores';
 import useAuthorize from './useAuthorize';
 
 const useGetPasskeysList = () => {
-    const { client } = useStore();
-    const { isSuccess } = useAuthorize();
+    const { client, common } = useStore();
+    const { isSuccess, isFetching: isAuthorizeFetching } = useAuthorize();
```

Comment:
why do we need isFetching here ?

Reply:
@Dhruv-deriv , because useAuthorize in such cases works incorrectly: isSuccess is true even when you switch off the connection and switch on again, so in case of reconnection we need to receive token, but if I just listen to network status and make a passkeys_list call I will be logged out, as authorize call wasn't sent before.
so we will not send any api calls until authorize is resent.
I think ideally we should do it in all places. :)

Reply:
cool thanks for the clarification 😄 LGTM in that case !